templates/base.html

```diff
@@ -3,13 +3,15 @@
 <html lang="en">
 
 <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
In HTML try to add the JS in `body` element if possible. That's the way to not block the page until the JS is loaded. I don't think jQuery is required while loading the page, so moving it to `body` shouldn't have any bad effect.